zos_basics/constructs/README.md

```diff
@@ -0,0 +1,100 @@
+# Job Submit Response Parsing and Constructs
+
+This sample focuses on processing the result of a z/OS® job submission response
+using a number of Ansible® constructs such as `when`, `with_sequence`, `loop`,
```

Comment:
can we hyperlink these contructs to the Ansible documentation pages for them?

Reply:
Done.